assets/templates/partials/calendar/items/list.tmpl

```diff
@@ -4,12 +4,22 @@
   tag designed for the purpose:
     <em class="ons-highlight">keyword</em>
 */}}
+{{ $currentPage := .Pagination.CurrentPage }}
+{{ $itemsPerPage := .Pagination.Limit }}
+{{ $totalSearchPosition := multiply (subtract $currentPage 1) $itemsPerPage }}
+
 <ol class="ons-list ons-list--bare ons-u-bt ons-u-mb-l">
-  {{range .Entries }}
+  {{range $i, $item := .Entries }}
+    {{ $currentPosition := add $i 1 }}
```

Comment:
Presumably you need to offset the index which is why you're adding `1` here?

Reply:
@mr-nick17 yes thats correct.